test/over_react/util/map_util_test.dart

```diff
@@ -253,5 +254,54 @@ main() {
         expect(newStyleFromProps(domProps()), equals({}));
       });
     });
+
+    group('getBackingMap()', () {
+      group('when passed in a props object', () {
+        JsBackedMap testMap = JsBackedMap.from({'id': 'test'});
+
+        var props = getBackingMap(TestProps(testMap));
```

Comment:
#nit Setup like this should go in a setUp/setUpAll, which allows:
- it not to be run when these tests aren't run due to filtering via command-line args
- any errors that could occur during setup to be associated properly with the tests

Reply:
That's super good to know, I wasn't aware of those benefits and thought that those utilities were primarily for DRY purposes. It's cool they do a lot